src/parser.js

```diff
@@ -5,6 +5,15 @@ const parser = require('solidity-parser-antlr');
 const parse = text => {
   const parsed = parser.parse(text, { loc: true, range: true });
   parsed.comments = extract(text);
+
+  // `code` and `codeStart` are different after prettifying, so the tests fail when they have AST_COMPARE enabled.
+  // Since we don't use those properties, we just remove them.
+  parsed.comments.forEach(comment => {
+    delete comment.code;
+    delete comment.codeStart;
+    return comment;
+  });
```

Comment:
This kind of code should go in a `massageAST` function so you don’t have to iterate over all the comments again when formatting with `AST_COMPARE` enabled.